extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -331,12 +331,40 @@ export class ProjectsController {
 		this.refreshProjectsTree();
 	}
 
+	public async delete(context: BaseProjectTreeItem): Promise<void> {
+		const project = this.getProjectFromContext(context);
+
+		const confirmationPrompt = context instanceof FolderNode ? constants.deleteConfirmationContents(context.friendlyName) : constants.deleteConfirmation(context.friendlyName);
+		const response = await this.apiWrapper.showQuickPick([constants.yesString, constants.noString].map(x => { return { label: x }; }), { placeHolder: confirmationPrompt });
```

Comment:
Is there a reason to use quickpick over `vscode.window.showWarningMessage()`? It would be nice if the delete experience looked the same as deleting a file in a folder that's open in the explorer

Reply:
Only reason was that I didn't know about the modal option!  Changed.